PhysicsTools/PatAlgos/plugins/JetCorrFactorsProducer.cc

```diff
@@ -232,12 +232,9 @@ JetCorrFactorsProducer::produce(edm::Event& event, const edm::EventSetup& setup)
 					    << "the sequence.";
     }
     for(unsigned int idx=0; idx<corrLevel->second.size(); ++idx){
-      bool flavorDependent=false;
       std::vector<float> factors;
-      if(flavorDependent ||
-	 corrLevel->second[idx].find("L5Flavor")!=std::string::npos ||
+      if(corrLevel->second[idx].find("L5Flavor")!=std::string::npos ||
```

Comment:
It looks like we need to either of remove "flavorDependent" because it does not seem to do anything or follow the suggestion from @cvuosalo if there is any intention that we apply the flavor dependent correction afterward for any corrections once the "L5Flavor" or "L7Parton" is encountered.